tests/documents.test.ts

```diff
@@ -140,6 +142,35 @@ describe('Documents tests', () => {
         expect(documents.offset).toEqual(2)
       })
 
+      test(`${permission} key: Get documents with filters`, async () => {
+        const client = await getClient(permission)
+        await client.index(indexPk.uid).updateFilterableAttributes(['id'])
+        const { taskUid } = await client
+          .index(indexPk.uid)
+          .addDocuments(dataset)
+        await client.waitForTask(taskUid)
+
+        const documents = await client.index(indexPk.uid).getDocuments<Book>({
+          filter: [['id = 1', 'id = 2']],
+        })
+
+        expect(documents.results.length).toEqual(2)
+      })
+
+      test(`${permission} key: Get documents should trigger error with a hint on unknown route`, async () => {
+        const apiKey = await getKey(permission)
+        const client = new MeiliSearch({ host: `${HOST}/indexes`, apiKey })
+
+        try {
+          await client.index(indexPk.uid).getDocuments({ filter: '' })
+          fail('it should not reach here')
```

Comment:
The other test in the `deleteDocuments` you used a different message, can you do the same here, or change the other one?